cl/search/documents.py

```diff
@@ -1058,6 +1061,12 @@ def prepare_trustee_str(self, instance):
     def prepare_plain_text(self, instance):
         return escape(instance.plain_text)
 
+    def prepare_cites(self, instance):
+        return [
+            opinion.cited_opinion.pk
+            for opinion in instance.cited_opinions.all()
```

Comment:
This is going to make a bunch of queries, isn't it? Is there a faster way to get the IDs we need?

Reply:
you're right, I've changed it to get the IDS in a single query:
`instance.cited_opinions.all().values_list("cited_opinion_id", flat=True)`

I also updated other prepare methods where this optimization is also possible.